releases/Process/Stabilization Process.md

```diff
@@ -0,0 +1,173 @@
+
+# Draft - O3DE Major Release Process
+V1.1 Updated December 22, 2022. 
+The release process is managed by the Release Special Interest Group (SIG). To request an update to this document, please open an issue at https://github.com/o3de/sig-release/issues
+
+## Key Details
+*Intent: Outlines the high-level information about O3DE releases.*
+
+* Releases are owned and managed by the Release SIG.
+* The Release SIG relies on the help of others in the O3DE community (outside of SIG-Release). See "Roles and Responsibilities" below
+* O3DE Major Releases occur twice each year: approximately in April and October. These dates are discussed in Release SIG meetings.
+* O3DE Minor Releases occur as needed, as decided by the Release SIG.
+* Releases are named  according to recommendations from the O3DE marketing committee as outlined at https://github.com/o3de/sig-release/issues/39
+* Each release managed using a a Github project board with the set of release process tasks.
+* Releases will update the following Repositories:
```

Comment:
Does this need to include o3de-extras?

Reply:
Good question. The Readme at https://github.com/o3de/o3de-extras doesn't really describe what that repo is for. If it's important to keep this repo synced and tagged for each release with the main O3DE repo, then maybe it makes sense. @amzn-changml thoughts on whether we need to include o3de-extras repo in our release process?

Reply:
I strongly suggest we not tie releases to the extras repo. The authors of those gems/resources are responsible for keeping those up to date, as everything on that repo is considered 'not critical' to the core of O3DE. We should send out a blast (email + social media) to announce release dates, giving authors time to update gems and resources as required.

Reply:
@forhalle In order to consider the o3de-extras repo sig-release needs to understand the argument for. If we can get an argument for, then @vincent6767  and I can discuss it in the next sig-release meeting.

Reply:
I'm tagging @byrcolin, as the architect of O3DE extras, to weigh in on this.

Reply:
Yes, going forward on our next release o3de/o3de-extras is considered part of the release. I will update the ReadMe on the repo to be more explicit on what is for.

Reply:
ok. This initial PR is to post our existing release process. I don't consider this blocking at the moment for this PR. In order for us to change the process we will need to discuss in our next meeting. @vincent6767 let's discuss in next week's sig-release meeting.

Reply:
Yes, for the next release, o3de-extras *should* be part of the process for transitioning into the workflow, but it currently does not block the current documentation of the process. @byrcolin as I understand, there's additional (planned) work to move the "required" gems into o3de-extras, but it currently contains optional components.

Reply:
https://github.com/o3de/sig-release/issues/124
@byrcolin @tonybalandiuk @amzn-changml @forhalle @Ulrick28 Created an agenda for the O3DEs extra for January 14th SIG release meeting.